etc/batfish-agent.json

```diff
@@ -0,0 +1,5 @@
+{
+    "namespace": "default",
+    "agent_class": "alto.agent.batfish.BatfishAgent",
+    "refresh_interval": 300
+}
```

Comment:
I wonder why the agent JSON file doesn't include any config of network control plane snapshot data.